.env

```diff
@@ -0,0 +1 @@
+POSTGRES_PASSWORD=postgrespassword
```

Comment:
I'm curious about this. let's chat about it.

Reply:
There are a couple of other patterns we could use for this, see: https://docs.docker.com/compose/environment-variables/

Reply:
great link, thanks!

I now see that this is a default and that system environment variables will override it. perfect.